pytorch_to_returnn/torch/nn/modules/operator.py

```diff
@@ -212,6 +212,23 @@ def create_returnn_layer_dict(self, *inputs: Tensor) -> Dict[str, Any]:
       "from": [self._get_input_layer_name(x) for x in inputs]}
 
 
+class Tile(Module):
+  """
+  Wraps RETURNN TileLayer.
+  """
+  is_original_torch_module = False
+
+  def __init__(self, multiples: Dict[str, int]):
+    super(Tile, self).__init__()
+    assert isinstance(multiples, dict)
+    self.multiples = multiples
+
+  def create_returnn_layer_dict(self, *inputs: Tensor) -> Dict[str, Any]:
+    return {
+      "class": "tile", "multiples": self.multiples,
+      "from": [self._get_input_layer_name(x) for x in inputs]}
```

Comment:
Why do you support having multiple inputs here?
A module should be simple, and not do extra logic.
```suggestion
  def create_returnn_layer_dict(self, input: Tensor) -> Dict[str, Any]:
    return {
      "class": "tile", "multiples": self.multiples,
      "from": self._get_input_layer_name(input)}
```